modules/repository-datastax/db/mutation/V030__Drop_User_Assignment_Look_Up_table.cql

```diff
@@ -0,0 +1 @@
+drop table user_assignment_look_up;
```

Comment:
Would it be better to use "DROP TABLE IF EXISTS keyspace_name.table_name" syntax?